core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -54,6 +78,20 @@ def get_gcs_resource_bucket_name():
         str. The bucket name for the application's GCS resources.
     """
     if constants.DEV_MODE:
-        return app_identity.get_default_gcs_bucket_name()
+        return get_default_gcs_bucket_name()
     else:
         return get_application_id() + _GCS_RESOURCE_BUCKET_NAME_SUFFIX
+
+
+def get_default_gcs_bucket_name(deadline=None):
```

Comment:
Don't pass in deadline if we don't need/use it.  Let's keep our version of the API as narrow as possible.

Reply:
Done.